.gitignore

```diff
@@ -1,2 +1,4 @@
 /vendor/
 composer.lock
+# IDE
+/.idea/
```

Comment:
yes please. 
For files/dirs related to your environment, such IDE files, please use a global gitignore.
If you use a global gitignore, for example in your home dir, it will be used in all your git repos.
For reference:
https://help.github.com/articles/ignoring-files/
https://davidwalsh.name/global-gitignore
https://git-scm.com/docs/gitignore (look for core.excludesFile)